jenkins/custom-ci/generic/cortx-rgw-integration.groovy

```diff
@@ -41,7 +47,15 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
 
                 sh encoding: 'UTF-8', label: 'cortx-provisioner', script: '''
-                bash ./jenkins/build.sh -v 2.0.0 -b ${CUSTOM_CI_BUILD_ID}
+                    if [ "${ENABLE_ADDB_PLUGIN}" == "yes" ]; then
+                        pushd ../
+                        git clone https://github.com/Seagate/cortx-motr.git && git clone https://github.com/Seagate/cortx-rgw
+                        popd
+                        bash ./jenkins/build.sh -v 2.0.0 -b ${CUSTOM_CI_BUILD_ID} -addb
+                    else
```

Comment:
I will Add the input parameter for the branch name and branch URL and export them here post successful testing.

Reply:
After discussing with @shailesh-vaidya , I am keeping the Git clone logic in the code

Reply:
> I will Add the input parameter for the branch name and branch URL and export them here post successful testing.

@Abhirup-A-G Please address this. Use repo_url and branch name provided for custom-ci input.

Reply:
Done